automation/override-bot/override-bot.py

```diff
@@ -0,0 +1,153 @@
+import requests, json, datetime, os
+from enum import Enum
+from github import Github
+
+ORG_NAME = 'kubevirt'
+REPO_NAME = 'hyperconverged-cluster-operator'
+GITHUB_BASE_API = 'https://api.github.com/repos'
+
+class Result(Enum):
+    Success =    0
+    Overridden = 1
+    Failure =    2
+    Pending =    3
+    Error   =    4
+    Aborted =    5
+    Invalid =    6
+
+class OverrideBot:
+    def __init__(self):
+        self.pr_list = []
+        self.start_time = datetime.datetime.now()
+        self.finish_time = None
+
+        github_token = os.environ['HCO_BOT_TOKEN']
+        gh = Github(github_token)
+        repo_name = f'{ORG_NAME}/{REPO_NAME}'
+        self.repoObj = gh.get_repo(repo_name)
+
+    def get_prs(self):
+        get_prs_req = requests.get(f'{GITHUB_BASE_API}/{ORG_NAME}/{REPO_NAME}/pulls')
+        pr_full_list = json.loads(get_prs_req.text)
+        for pr in pr_full_list:
+            if True or 'do-not-merge/hold' not in [label['name'] for label in pr['labels']]:
+                prObj = PullRequest(pr['number'], pr['title'], pr['url'], pr['_links']['statuses']['href'])
+                self.pr_list.append(prObj)
+
+    def get_ci_tests(self):
+        for pr in self.pr_list:
+            pr.get_ci_tests()
+
+    def nominate_lanes_for_override(self):
+        for pr in self.pr_list:
+            pr.nominate_lanes_for_override()
+
+    def comment_overrides(self):
+        for pr in self.pr_list:
+            pr.comment_overrides(self.repoObj.get_pull(pr.number))
+
+class PullRequest:
+    def __init__(self, number, title, gh_url, statuses_url):
+        self.number = number
+        self.title = title
+        self.gh_url = gh_url
+        self.statuses_url = statuses_url
+        self.ci_tests_list = []
+        self.override_list = []
+
+    def get_ci_tests(self):
+        statuses_raw = requests.get(self.statuses_url).text
+        statuses = json.loads(statuses_raw)
+        for status in statuses:
+            context = status['context']
+            if 'ci-index' in context or 'images' in context or 'prow' not in context:
+                continue
+            splitted = context.split('/')[-1].split('-')
+            provider = splitted[-1]
+            test_name = '-'.join(splitted[:-1])
+            state = status['state']
+            overridden = True if status['description'] and 'Overridden' in status['description'] else False
```

Comment:
A Boolean value is just a result of a Bolean expression. no need to explicitly set True and False; e.g.
overridden = status['description'] and 'Overridden' in status['description']
```python

```

Reply:
Thanks for pointing that out. I'll change in a following PR.